Makefile

```diff
@@ -18,4 +18,10 @@ check:
 
 all: functional
 
+checkcommits:
+	cd pkg/checkcommits/ && go build && go test
```

Comment:
Can we add the tests to the usual make check? Sounds like a good idea to execute them at every commit once we've hooked CI to this repo